buildflow/core/app/flow.py

```diff
@@ -342,20 +342,25 @@ async def teardown(self):
                 include_sink_primitive = True
 
             def pulumi_resources():
-                source_resources = []
+                all_resources = []
+                # Builds the source's pulumi.CompositeResource (if it exists)
                 if source_primitive.managed and include_source_primitive:
```

Comment:
Im wondering if we should move all references to managed inside the pulumi() method now. If its not managed, it would return None and would be handled on line 353

Reply:
I think we still need a managed reference in the primitive so the user can set it. But we that could just be passed through to the provider